fairseq/models/masked_lm.py

```diff
@@ -282,7 +282,7 @@ def base_bert_architecture(args):
     args.no_bias_kv = getattr(args, 'no_bias_kv', True)
```

Comment:
Also can you rename this function to `bert_base_architecture`. It doesn't actually change anything, but is more semantically correct.